inductiva/tasks/task.py

```diff
@@ -155,3 +156,22 @@ class _PathParams(TypedDict):
 
     def _get_path_params(self) -> _PathParams:
         return {"task_id": self.id}
+
+    def get_time(self) -> float:
+        """Get the time the task took to complete.
+
+        Returns:
+            The time in seconds.
+        """
+
+        if self.get_status() != TaskStatusCode.SUCCESS:
```

Comment:
Shall this be just for successful tasks? I think this may also be useful for other events.